src/main/Config.cpp

```diff
@@ -217,6 +217,8 @@ Config::Config() : NODE_SEED(SecretKey::random())
 
     HISTOGRAM_WINDOW_SIZE = std::chrono::seconds(30);
 
+    HALT_ON_INTERNAL_TRANSACTION_ERROR = false;
```

Comment:
need to update example config, this is how we test that parsing is working properly

Reply:
Updated the config. I've been thinking about updating it after this change is in to prevent the mismatch between docs and binaries, but I suppose the scenario of someone just copying the example config at head is unlikely.